### DISCUSSION TITLE
Is it possible to make a mobile app or some way to automate the sending of links to our bookmark?

### DISCUSSION THREAD

**Discussion by mapachitodev** [goniszewski/grimoire#147] [General]
My favourite app is dead, so I need a new system. I've been looking at this as the best alternative but I'm held back by the lack of a mobile app or anything that can help me. Any ideas on how we can contribute to the development of this?

**Comment by goniszewski** [comment]
Hello @mapachitodev! If you're using an iPhone, you can create a shortcut based on [this example](https://github.com/goniszewski/grimoire/issues/114). I don't know what possibilities are there on Android, though.

PWA support is planned for Grimoire just after the introduction of import features and updated gen AI integration.
